### DISCUSSION TITLE
Not yet received the Education benefits pack. It was approved on 6th Nov 2024

### DISCUSSION THREAD

**Discussion by judynicky** [community/community#144299] [GitHub Education]
### Hi everyone,

I applied for Github Education. It says approved however still not received the benefits pack or any emails to my university email or personal. It was approved on 6th Nov 2024. Please assist
![image](https://github.com/user-attachments/assets/6bc138ef-1621-420e-bb05-090d676aa267)

**Comment by davevad93** [comment]
Hi @judynicky ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @judynicky was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by judynicky** [reply]
Make sense. I'll wait. May be I need to wait for another week.

**Comment by YashG2009** [comment]
Did you recieve now? Just asking to know because mine has also taken lot of time.

**Reply by judynicky** [reply]
Not yet, I received the approval 6 days ago
